doc/symfony6.md

```diff
@@ -0,0 +1,445 @@
+# Install Gedmo Doctrine extensions in Symfony 6
```

Comment:
How is the integration with Symfony 6 related to the deprecation fix for `EventArgs::getEntityManger()`?

Reply:
It's not related to EntityManager, it's related to the deprecation of Event Subscribers. This comes from the Symfony Bridge: https://github.com/symfony/symfony/issues/49586

Reply:
The simpler solution here might be to make the documentation link to the bundle at https://packagist.org/packages/stof/doctrine-extensions-bundle, which handles registering listeners instead of subscribers, with CI checks warning us if the events used by a listener are updated.